src/ui/public/vislib/visualizations/point_series/column_chart.js

```diff
@@ -10,8 +10,11 @@ export function VislibVisualizationsColumnChartProvider(Private) {
     mode: 'normal',
     showTooltip: true,
     color: undefined,
-    fillColor: undefined
+    fillColor: undefined,
   };
+
+  const minBarWidth = 1;
```

Comment:
actually i am having second thoughts about this one ... 
should we even allow that ? (to do split chart with 50 charts ?) ... 
maybe the right way to fix this would be to show more meaningful error message in this case ? 

"Splitting charts is limited to 5 charts" or something like this ?